test/cli/repo.js

```diff
@@ -16,4 +16,14 @@ describe('repo', () => runOnAndOff((thing) => {
     const out = await ipfs('repo version')
     expect(out).to.eql(`${repoVersion}\n`)
   })
+
+  it('get repo stats', async () => {
+    const stats = await ipfs('repo stat')
+    expect(stats).to.not.include('MB').and.not.include('GB')
```

Comment:
Could include `KB` or `TB`?

This might be better if you parse the repo size and assert that it's an integer?  Or use a regex if we want [two problems](http://regex.info/blog/2006-09-15/247)?

```javascript
expect(stats).to.match(/^\s+repo\ssize:\s+\d+$/gm)
```

Reply:
Nope, it can only include `MB` and `GB`. I think regex will work fine in this case.

Reply:
What I mean is someone could change the implementation to pretty-print the sizes in 'KB' and as implemented this test would still pass.